js_modules/dagster-ui/packages/ui-components/src/components/CollapsibleSection.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import {Box} from './Box';
+import {Icon} from './Icon';
+
+export const CollapsibleSection = ({
+  header,
+  headerWrapperProps,
+  children,
+  isInitiallyCollapsed = false,
+}: {
+  header: React.ReactNode;
+  headerWrapperProps?: React.ComponentProps<typeof Box>;
+  children: React.ReactNode;
+  isInitiallyCollapsed?: boolean;
+}) => {
+  const [isCollapsed, setIsCollapsed] = React.useState(isInitiallyCollapsed);
```

Comment:
Hmm does this one look like the new Asset Overview one? (the "Description" header here: https://www.figma.com/file/wQHGZ2cjfpna2P2OaM4aTs/aset-catalog-ui?type=design&node-id=208-45868&mode=design&t=IUz5OJkO0QRbYlOv-0)

Maybe we can merge this one and that together.  I'd left it in the AssetNodeOverview.tsx file because there weren't any other collapsing sections like that in the product, but maybe there are now!